src/components/Asset/AssetContent/MetaMain/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement } from 'react'
+import styles from './index.module.css'
+import { Asset } from '@oceanprotocol/lib'
+import { decodeTokenURI } from '@utils/nft'
+import MetaAsset from './MetaAsset'
+import MetaInfo from './MetaInfo'
+import Tooltip from '@shared/atoms/Tooltip'
+import NftTooltip from './NftTooltip'
+import Logo from '@shared/atoms/Logo'
+
+export default function MetaMain({ ddo }: { ddo: Asset }): ReactElement {
+  const nftMetadata = decodeTokenURI(ddo?.nft?.tokenURI)
+
+  const blockscoutNetworks = [1287, 2021000, 2021001, 44787, 246, 1285]
+  const isBlockscoutExplorer = blockscoutNetworks.includes(ddo?.chainId)
```

Comment:
why do we have this check?

Reply:
I copied this implementation from the previous `MetaMain` code. It basically helps to decide whether to use `{explorer}/tokens` or `{explorer}/token` to link to the nft / datatoken respectively.
See line 52 of src/components/Asset/AssetContent/MetaMain/NftTooltip.tsx for example
```ts
isBlockscoutExplorer ? `tokens/${address}` : `token/${address}`
```

Reply:
i see, thanks